contrib/rabbitmq/cmd/receive_adapter/main.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"flag"
+	"github.com/knative/eventing-sources/contrib/rabbitmq/pkg/adapter"
+	"github.com/knative/pkg/signals"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"log"
+	"os"
+	"strconv"
+)
+
+const (
+	envBrokers             = "RABBITMQ_BROKERS"
```

Comment:
This that mine you cannot pass user and password as a secret? We should change that.

Reply:
Hello @Justin2997, thanks for your comment. I'll work on this.

Reply:
Let me know if you need help on that.

Reply:
Hi @Justin2997, thanks for your comment. you can work on this.

Reply:
Ok, I will try to make it work before Monday. Thanks